src/components/PostDetails/PostDetails.scss

```diff
@@ -0,0 +1,58 @@
+.PostDetails {
+  &__post {
+    padding: 10px 0 15px;
+  }
+
+  &__comments {
+    text-align: right;
+  }
+
+  &__visiblelist {
+    display: none;
+  }
+
+  &__list {
+    list-style: none;
+  }
+
+  &__list-item {
+    margin: 5px 0;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 10px;
+    border-radius: 10px;
+    font-style: italic;
+    font-weight: 300;
+    font-size: 0.9rem;
+    background: linear-gradient(
+        to right, rgba(255,255,255,1) 0%,
+        rgba(246,246,246,1) 47%,
+        rgba(250,250,253,1) 100%
```

Comment:
Tip: Insert this gradient to a separated scss file and use it in here and any other place that uses this specific gradient.